portal/add_edit_event_user.php

```diff
@@ -527,6 +482,15 @@
 
 if ($_POST['form_action'] != "") {
   // Leave
+    $type = $insert ? xlt("A New Appointment") : xlt("An Updated Appointment");
+    $note = $type . " " . xlt("request was received from portal patient ");
+    $note .= text($_SESSION['ptName']) . " " . xlt("regarding appointment dated ") . text($event_date) . " " . text($starttime) . ". ";
+    $note .= !empty($_POST['form_comments']) ? (xlt("Reason ") . text($_POST['form_comments'])) : "";
+    $note .=  ". " . xlt("Use Portal Dashboard to confirm with patient.");
+    $title = xlt("Patient Reminders");
```

Comment:
remove all the html escaping from above since you should not be html escaping stuff that goes to database. it will get html escaped after it is pulled from database.
:)

Reply:
also shouldn't have any ending whitespace in the constants. pasted below the fixes
```
    $type = $insert ? xl("A New Appointment") : xl("An Updated Appointment");
    $note = $type . " " . xl("request was received from portal patient") . " ";
    $note .= $_SESSION['ptName'] . " " . xl("regarding appointment dated") . " " . $event_date . " " . $starttime . ". ";
    $note .= !empty($_POST['form_comments']) ? (xl("Reason") . " " . $_POST['form_comments']) : "";
    $note .=  ". " . xl("Use Portal Dashboard to confirm with patient.");
    $title = xl("Patient Reminders");
```

Reply:
lol, completely forgot I was writing to database but what would a PR be from me if I wasn't over escaping or not escaping:)